weareinreach/inreach-catalog
# refactors files to support localization

## Description

Refactors the following:
- MoreMobile.js
- AccountMobile.js
- LoginFormContainer.js
- LoginForm.js

- Asana ticket: Related to #117 

## PR Checklist




## How to Test

- Launch the app (mobile view)
- Click "More"
   - Verify all sections display correctly
   - Verify all links function correctly
- Click "Account"
   - Verify all sections display correctly
   - Verify all links function correctly


### Regression Checks:
 - Existing user is able to sign in
 - New user is able to sign up
 - Tab toggle between Sign In and Sign Up screens function correctly
